.changeset/big-crabs-tap.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: regex support for `ignorePseudoClasses` option of `selector-pseudo-class-no-unknown`
```

Comment:
[ask] Is it no problem to consider this PR a *minor* change? Or a bug fix?

Reply:
It's not clear-cut. I think minor is fine.